PrimaryInstallScript.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+
+ping -q -c1 google.com > /dev/null
+ 
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#install SimpleAmqpClient
+git clone https://github.com/alanxz/SimpleAmqpClient /tmp/SimpleAmqpClient/
+mkdir /tmp/SimpleAmqpClient/build
+(cd /tmp/SimpleAmqpClient/build && cmake \
+	--yes --force-yes \
+	-DRabbitmqc_INCLUDE_DIR=../../rabbitmq-c/librabbitmq \
+	-DRabbitmqc_LIBRARY=../../rabbitmq-c/build/librabbitmq ..)
+(cd /tmp/SimpleAmqpClient/build make)
+mkdir /usr/local/include/SimpleAmqpClient
+cp *.so* /usr/local/lib/
+cp ../src/SimpleAmqpClient/*.h /usr/local/include/SimpleAmqpClient
+ldconfig -v
+
+#install Epsilon-Hermes
```

Comment:
We also need to add in the installation of the google test suite, and add the gmock/gtest folders to `/usr/local/include`